internal/pkg/daemon/enricher/enricher.go

```diff
@@ -27,23 +30,39 @@ import (
 	"sigs.k8s.io/security-profiles-operator/internal/pkg/config"
 )
 
+const defaultCacheTimeout time.Duration = 6 * time.Hour
```

Comment:
Given the ephemeral nature of containers, maybe we should choose a lower cache timeout. e.g. only a couple of hours maybe? or even less, what do you think? We should document our reasoning in comments here.

Reply:
Yeah I think we should reduce it, I've chosen 1 hour. I also added a comment that it's nothing more than a rough guess :man_shrugging:

Reply:
Right, at least now people won't wonder why a value was chosen. Maybe one day we'll have enough usage and data for the operator to choose a value based on data :)